prow/plugins/assign/assign.go

```diff
@@ -134,13 +157,39 @@ func handle(gc githubClient, log *logrus.Entry, ae assignEvent) error {
 	if len(unassign) > 0 {
 		log.Printf("Removing assignees from %s/%s#%d: %v", ae.org, ae.repo, ae.number, unassign)
 		if err := gc.UnassignIssue(ae.org, ae.repo, ae.number, unassign); err != nil {
-			return err
+			msg := fmt.Sprintf("Unassign failed: %v.", err.Error())
```

Comment:
We don't post a comment every time something goes wrong. If `UnassignIssue` fails then something is wrong in prow, or maybe GitHub is down. This should be rare, and we don't post comments in this case.